plugins/reolink/src/reolink-api.ts

```diff
@@ -73,25 +73,30 @@ export class ReolinkCameraClient {
         return pt;
     }
 
-    async login() {
-        if (this.tokenLease > Date.now()) {
+    async login(forceToken?: boolean) {
+        if ((!forceToken ? true : this.tokenLease !== Infinity) && this.tokenLease > Date.now()) {
```

Comment:
Add check, if the token is forced, make sure it's not Infinity otherwise it would always pass if the previous login was not forced